work_items/vc_marketplace.md

```diff
@@ -0,0 +1,103 @@
+
+(Mechanics of a real world credential exchange (protocol) covering value exchange can be defined to establish the reference architecture for a Verifiable Credentials Marketplace.)
+
+## Deliverables
+
+High level architecture for Verifiable Credentials Marketplace independent from solution architecture / protocol design that covers the following items:
+
+- Use cases of VC Marketplace
+- VC metadata required to enable the Marketplace
+- Discovery mechanisms:
+    - Who are the issuers of credentials that I need?
+    - What credentials do I need to receive service?
+- Aggregated and anonymous tracking of VC Marketplace activity
+- Privacy, security, and compliance considerations across marketplace actors
+
+The group believes that the VC Marketplace can be separated into three distinct parts around Discovery, (Credential) Exchange, and Payments. In the first iteration the group will concentrate on the Discovery aspect.
+
+**Part 1: Discovery of Credential**
+
+- Start with mission statement of this work and user journeys for several commercial use cases
+    - Definition of Credential Metadata
+    - Discovery of Interaction Partners by Credential Metadata
+- How can this metadata be captured?
+    - Leverage Credentials Manifest + Presentation Exchange and build upon
+    - Issuer Manifest — specifying types of VCs that can be requested
+    - Verifier Manifest — specifying types of VCs that are required to get services
+    - Credential Manifest — VC metadata
+- Some Example User Stories that the group will design:
+    - Issuer: How to communicate business and technical credential requirements to prospective holders?
+        - What Issuers do I accept?
+        - What Compliance Level do I need (these could have technical implications)
+            - What Regulation does the Credential need to adhere to?
+        - Holder:
+            - How to discover parties that accept my credentials?
+            - Where can I use these credentials to avail services?
+            - Where can I acquire credentials to get access to service X?
+        - Verifiers / RP:
+            - If holders are missing valid credentials to consume my service, how are holders redirected to a matching issuer. How much user-control choice? How is (unverified) subject-related data forwarded back into the issuance process.
+            - There should NOT be a strong coupling between Verifier and Issuer in this process
+
+**Part 2: Credential Exchange** (Out of scope for the first iteration)
+
+- Credential exchange - separate category
+- Value exchange - separate scope/category
+
+**Part 3: Payments** (Out of scope for the first iteration)
+
+- Payments - combined scope with Value exchange
+
+Expected timeline
+
+- 4 months: January 2021 — April 2021
+
+Audience of the deliverables
```

Comment:
@Gerstep Can we add the meeting time (weekly 8am-9am ET) here as well. (can also be done later)

Reply:
@rado0x54 I didn't get an invite. What day of the week are going with?